### DISCUSSION TITLE
Can't reset my password

### DISCUSSION THREAD

**Discussion by pgbsean** [mastodon/mastodon#22681] [Q&A]
Is your server down or sth? I forgot the password to my Mastodon account, and now I want to reset it, but your forgot my password website won't even send out the email to me.

**Comment by rbairwell** [comment]
Hi, this Github repository is more for problems running the code/software rather than end-user support, but I'll try and help.

Each Mastodon server (also called an instance) is individually run and only mastodon.social and mastodon.online are actually operated by Mastodon gGmbh. Your Mastodon username will be in the format @username@instance.tld : if you go to https://instance.tld/ and tru and sign in you should be able to request a new password- it may take a short time to come through depending on the volunteer run servers backlog.

Can't remember which server/instance you picked? Search your email for the "Welcome to Mastodon" email and it'll have your full Mastodon username/handle in the "Your full handle" section.

If you need further help, we'll need your full handle/username to be able to direct you to the correct instance or contact that instances server admin.

**Comment by pgbsean** [comment]
Thank you! It worked for me!
